server/src/config/configs.js

```diff
@@ -72,4 +48,35 @@ export const isTeacher = async (req, res, next) => {
     }
   }
   return res.status(401).json({ error: "Not authorized" });
-};
\ No newline at end of file
+};
+
+const storage = multer.diskStorage({
+  destination: (req, file, cb) => {
+    const uploadPath = path.join(__dirname, '../../public/students_CV');
```

Comment:
if i am not wrong, the folder public is not okay. It should be another one.
I will provide you more info in following comments.